### ISSUE TITLE
[Bug]: Address search returns shifted results with EPSG:3003

### ISSUE THREAD

**Issue by RobiFag** [3liz/lizmap-web-client#4751]
### What is the bug? (in English)

I have a [QGIS project](https://map.gishosting.eu/index.php/view/map?repository=gter_full&project=progetto_nominatim_gter) with reference system EPSG:3003, using nominatim the address search returns shifted results. Changing the project reference system (we tried several EPSG 4326, 3857, 32632, 25832, 7791, ecc.) it works correctly.

Maybe something related to this issue #1624 ?

![image](https://github.com/user-attachments/assets/ffde44ad-802f-4c7c-8117-82f50f6d3d77)

### Steps to reproduce the issue

1. Create a project with reference system EPSG:3003
2. add some data
3. activate the address search bar with nominatim from lizmap plugin
4. search for an address using the web tool

### Versions, safeguards, check summary etc

**Versions :**

* Lizmap Web Client : 3.7.9
* Lizmap plugin : 4.3.15
* QGIS Desktop : 3.34.4
* QGIS Server : 3.34.10
* Py-QGIS-Server : not used
* QGIS Server plugin atlasprint : 3.4.1
* QGIS Server plugin lizmap_server : 2.9.4
* QGIS Server plugin wfsOutputExtension : 1.8.0

<details>
<summary>List of safeguards :</summary>
<br/>

* Mode : normal
* Allow parent folder : no
* Prevent other drive : yes
* Prevent PG service : no
* Prevent PG Auth DB : yes
* Force PG user&pass : yes
* Prevent ECW : yes
</details>

### Check Lizmap plugin

- [X] I have done the step just before in the Lizmap QGIS desktop plugin before opening this ticket. Otherwise, my ticket is not considered valid and might get closed.

### Operating system

Ubuntu 20.04.6 LTS

### Browsers

Firefox, Chrome, Microsoft Edge

### Browsers version

Firefox Versione 130.0, Chrome Versione 128.0.6613.138, Microsoft Edge Versione 128.0.2739.79

**Comment by RobiFag** [comment]
I have just updated the lizmap plugin but nothing changed. At the moment EPSG:3003 and EPSG:3004 are the only ones we have encountered the problem with.

Regarding testing LWC 3.8.1, we need some time but I think we can make some tests.

**Comment by Gustry** [comment]
> I have just updated the lizmap plugin but nothing changed. 

Yes, it wasn't related in this case, but always good to upgrade ;-)

**Comment by RobiFag** [comment]
Hi, I have already tested LWC 3.8.0 and the issue is still there, moreover the small yellow dot does not appear after searching.

**Comment by Gustry** [comment]
Yes, it looks linked to #1624
